tools/certs/ee-p384.certspec

```diff
@@ -1,6 +1,6 @@
 issuer:int-p256
-subject:ee-p256
-issuerKey:secp256r1
+subject:ee-p384
+issuerKey:secp384r1
 subjectKey:secp384r1
 signature:ecdsaWithSHA384
```

Comment:
I think this is what's wrong - this certificate, though it has a secp384 key, should be signed with ecdsaWithSHA256.